src/cleaner/cleaner_main.py

```diff
@@ -76,31 +76,20 @@ def sync(self):
             time.sleep(1)
 
 
-def get_worker(arg):
-    if arg == "docker_cache":
-        worker = Worker(clean_docker_cache.check_and_clean, 50, timeout=timedelta(minutes=10), cool_down_time=1800)
-    elif arg == "deleted_files":
-        worker = Worker(check_deleted_files.list_and_check_files, None, timeout=timedelta(minutes=10), cool_down_time=1800)
-    else:
-        raise ValueError("arguments %s is not supported.", arg)
-    return worker
-
-
-liveness_files = {
-    "docker_cache": "docker-cache-cleaner-healthy",
-    "deleted_files": "deleted-files-cleaner-healthy"
-}
+def get_worker(threshold):
+    worker = Worker(clean_docker.check_and_clean, threshold, timeout=timedelta(minutes=10), cool_down_time=60)
+    return worker;
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("option", help="the functions currently supported: [docker_cache | deleted_files]")
+    parser.add_argument("-t", "--threshold", help="the disk usage precent to start cleaner")
     args = parser.parse_args()
 
     common.setup_logging()
 
-    cleaner = Cleaner(liveness_files[args.option])
-    cleaner.add_worker(args.option, get_worker(args.option))
+    cleaner = Cleaner("docker-cleaner-healthy")
```

Comment:
Could we move this liveness file path to `clean_docker`, less hard-code here seems to be better